packages/common/src/utils/array.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ *
+ * Interleave array elements
+ *
+ * interleave ([0, 2, 4, 6], [1, 3, 5]) // [ 0 1 2 3 4 5 6 ]
+ * interleave ([0, 2, 4], [1, 3, 5, 7]) // [ 0 1 2 3 4 5 7 ]
+ * interleave ([0, 2, 4], [])           // [ 0 2 4 ]
+ * interleave ([], [1, 3, 5, 7])        // [ 1 3 5 7 ]
+ * interleave ([], [])                  // [ ]
+ */
+export const interleave = <A, B>([x, ...xs]: A[], ys: B[] = []): (A | B)[] =>
```

Comment:
ooh this is a neat one

Reply:
my initial thought what that this was also sorting the numbers so a comment there may be nice, but not needed for sure

Reply:
Oh yup the example is a little confusing, I can update in next pr!